File: programmers/level1/009.py
# 9. 문자열을 정수로 바꾸기
def solution(s):
    answer = 0
    if s[0]=='+':
        answer += int(s[1:])
    elif s[0]=='-':
        answer -= int(s[1:])
    else:
        answer = int(s)
    return answer

File: programmers/level2/009.java
// 9. 카펫
class Solution {
    public int[] solution(int brown, int yellow) {
        final int TOTAL = brown + yellow;
        int[] answer = new int[2];
        for(int i=1; i< TOTAL; i++){
            if(yellow%i==0){
                if( (yellow/i+2)*(i+2)==TOTAL){
                    answer[0]=TOTAL/(i+2);
                    answer[1]=i+2;
                    break;
                }
            }
        }
        return answer;
    }
}